### PR TITLE
Attribute docstrings support

### PR DESCRIPTION
**Pull request by mauvilsa**
Implements #71.

### CODE REVIEW

**Review by rr- — COMMENTED** [review]
LGTM, also the API seems OK now that I see the tests, but I have one remark: let's use imperative mood in the docstrings, so instead:
`"""If X happens, returns Y."""`
let's use:
`"""Return Y if X happens."""`

**Comment by mauvilsa** [comment]
> let's use imperative mood in the docstrings, so instead: `"""If X happens, returns Y."""` let's use: `"""Return Y if X happens."""`

Done.

File: docstring_parser/attrdoc.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    # Support simple cases in Python < 3.9
+    if isinstance(node, (ast.Str, ast.Num, ast.NameConstant, ast.Constant)):
+        return str(ast_get_constant_value(node))
+    if isinstance(node, ast.Name):
+        return node.id
+    return None
+
+
+def ast_is_literal_str(node: ast.AST) -> bool:
+    """Checks whether an AST node is a literal string."""
```
Comment (rr-) [inline docstring_parser/attrdoc.py:41]: Hi, sorry for the delay, I was on vacation. We still use descriptive mood in the docstrings here and other functions.

Comment (mauvilsa) [reply docstring_parser/attrdoc.py]: I updated the docstrings. Have a look to see if they are correct now.

**Review by rr- — APPROVED** [review]

**Pull request merged**